src/themes/default/components/core/blocks/Header/Header.vue

```diff
@@ -45,12 +48,13 @@
           </div>
           <div class="col-xs-5 col-md-3 end-xs">
             <div>
-              <a v-if="!currentUser" href="#" @click.prevent="gotoAccount" class="cl-tertiary links">
-                {{ $t('Login to your account') }}
-              </a>
-              <span v-else>
-                {{ $t('You are logged in as') }} {{ currentUser.firstname }}
-              </span>
+              <a
```

Comment:
I'm not sure if this formating is easier to read compared to the previous one. Maybe a new line or two would help :)